dart/test/sentry_client_test.dart

```diff
@@ -352,47 +352,75 @@ void main() {
   });
 
   group('SentryClient : apply partial scope to the captured event', () {
-    var options = SentryOptions(dsn: fakeDsn);
-    var scope = Scope(options);
-
-    final transaction = '/test/scope';
-    final eventTransaction = '/event/transaction';
-    const fingerprint = ['foo', 'bar', 'baz'];
-    const eventFingerprint = ['123', '456', '798'];
-    final user = SentryUser(id: '123');
-    final eventUser = SentryUser(id: '987');
-    final crumb = Breadcrumb(message: 'bread');
-    final eventCrumbs = [Breadcrumb(message: 'bread')];
-
-    final event = SentryEvent(
-      level: SentryLevel.warning,
-      transaction: eventTransaction,
-      user: eventUser,
-      fingerprint: eventFingerprint,
-      breadcrumbs: eventCrumbs,
-    );
+    late Fixture fixture;
 
     setUp(() {
-      options = SentryOptions(dsn: fakeDsn);
-      options.transport = MockTransport();
-      scope = Scope(options)
-        ..user = user
-        ..transaction = transaction
-        ..fingerprint = fingerprint
-        ..addBreadcrumb(crumb);
+      fixture = Fixture();
     });
 
     test('should not apply the scope to non null event fields ', () async {
-      final client = SentryClient(options);
-      await client.captureEvent(event, scope: scope);
+      final transport = MockTransport();
+      final client = fixture.getSut(true, transport);
```

Comment:
```suggestion
      final client = fixture.getSut(true, fixture.transport);
```

should not this be part of the fixture?

Reply:
true